### PR TITLE
docs(kitex): translate kitex-fallback doc based on Chinese doc

### PR DESCRIPTION
**Pull request by morpeko7**
#### What type of PR is this?

docs
#### What this PR does / why we need it (en: English/zh: Chinese):

en: translate kitex-fallback doc based on Chinese doc
zh:

#### Which issue(s) this PR fixes:

cloudwego/kitex#855

### CODE REVIEW

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md [hunk 1]
```diff
@@ -6,4 +6,216 @@ keywords: ["Kitex", "Fallback"]
...
+> Thrift : No specific requirement for the generated code version
+
+## 1.Kitex Fallback Functional Description
+
+
+After an RPC request fails, businesses usually take some fallback measures to ensure a valid response (such as constructing a default response when encountering timeouts or circuit breakers). Kitex's Fallback supports handling all exceptional requests. Additionally, as business exceptions usually return through Resp (BaseResp), Fallback also supports handling Resp.
+### 1.1 Result types that support Fallback
+
+1. **RPC** **Error**: RPC request exceptions such as timeout, circuit breaker, rate limiting, and protocol errors at the RPC level.
+2. **Business Error**: Business Custom Exceptions distinct from RPC Exception,Specifically [Kitex - 业务异常处理使用文档](../bizstatuserr)
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/basic-feature/fallback.md:22]: Kitex - Business Exception Usage Guide

Comment (morpeko7) [reply content/en/docs/kitex/Tutorials/basic-feature/fallback.md]: done

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md [hunk 2]
```diff
@@ -6,4 +6,216 @@ keywords: ["Kitex", "Fallback"]
...
+
+// use demo
+client.WithFallback(
+fallback.NewFallbackPolicy(
+func(ctx context.Context, args utils.KitexArgs,
+result utils.KitexResult, err error) (fbErr error) {
+// your fallback logic...
+result.SetSuccess(yourFallbackResult)
+return
+}))
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/basic-feature/fallback.md:84]: this code is not formatted

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md [hunk 3]
```diff
@@ -6,4 +6,216 @@ keywords: ["Kitex", "Fallback"]
...
+// RealReqRespFunc is the definition for fallback func with real rpc req as param, and must return the real rpc resp.
+// setup eg: client.WithFallback(fallback.NewFallbackPolicy(fallback.UnwrapHelper(yourRealReqRespFunc)))
+type RealReqRespFunc func(ctx context.Context, req, resp interface{}, err error) (fbResp interface{}, fbErr error)
+
+// use demo
+client.WithFallback(fallback.NewFallbackPolicy(fallback.UnwrapHelper(func(ctx context.Context, req, resp interface{}, err error) (fbResp interface{}, fbErr error) {
+// your fallback logic...
+return fbResp, fbErr
+}))
+```
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/basic-feature/fallback.md:103]: pls format all the code mentioned in this doc

Comment (morpeko7) [reply content/en/docs/kitex/Tutorials/basic-feature/fallback.md]: The code has been formatted, please take a look again.

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md [hunk 4]
```diff
@@ -6,4 +6,233 @@ keywords: ["Kitex", "Fallback"]
...
+xxxCli := xxxservice.NewClient("target_service", opts...)
+```
+
+### Call-level configuration
+
+```Go
+import (
+"github.com/cloudwego/kitex/pkg/retry"
+bretry "code.byted.org/kite/kitex/pkg/retry"
+)
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/basic-feature/fallback.md:48]: import 里面的内容需要缩进一下

Comment (morpeko7) [reply content/en/docs/kitex/Tutorials/basic-feature/fallback.md:48]: 好，还有其他要改的地方吗，我一起改一下

Comment (GuangmingLuo) [reply content/en/docs/kitex/Tutorials/basic-feature/fallback.md:48]: 我这没有了

**Comment by GuangmingLuo** [comment]
@YangruiEmma PTAL

**Review by GuangmingLuo — DISMISSED** [review]

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md [hunk 5]
```diff
@@ -6,4 +6,233 @@ keywords: ["Kitex", "Fallback"]
 description: Kitex Fallback Introduction and Usage Guide.
 ---
 
-Translation TODO 
+> **Support Version: >= v0.5.0  （go.mod dependencies: [github/cloudwego/kitex](https://github.com/cloudwego/kitex))**
+>
+> Protobuf Generated code version：>=v0.5.0（The version can be found in the header of the generated code file and in the version comments.）
```
Comment (YangruiEmma) [inline content/en/docs/kitex/Tutorials/basic-feature/fallback.md:11]: please use English colon  'version：>=v0.5.0'

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md [hunk 6]
```diff
@@ -6,4 +6,233 @@ keywords: ["Kitex", "Fallback"]
...
+> Thrift : No specific requirement for the generated code version
+
+## 1.Kitex Fallback Functional Description
+
+
+After an RPC request fails, businesses usually take some fallback measures to ensure a valid response (such as constructing a default response when encountering timeouts or circuit breakers). Kitex's Fallback supports handling all exceptional requests. Additionally, as business exceptions usually return through Resp (BaseResp), Fallback also supports handling Resp.
+### 1.1 Result types that support Fallback
+
+1. **RPC** **Error**: RPC request exceptions such as timeout, circuit breaker, rate limiting, and protocol errors at the RPC level.
+2. **Business Error**: Business Custom Exceptions distinct from RPC Exception,Specifically [Kitex - Business Exception Usage Guide](../bizstatuserr)
```
Comment (YangruiEmma) [inline content/en/docs/kitex/Tutorials/basic-feature/fallback.md:22]: Business Custom Exceptions, which is distinct from RPC Exception, for the details please see [Kitex - Business Exception Usage Guide](../bizstatuserr)

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md [hunk 7]
```diff
@@ -6,4 +6,233 @@ keywords: ["Kitex", "Fallback"]
...
+> **Support Version: >= v0.5.0  （go.mod dependencies: [github/cloudwego/kitex](https://github.com/cloudwego/kitex))**
+>
+> Protobuf Generated code version：>=v0.5.0（The version can be found in the header of the generated code file and in the version comments.）
+>
+> Thrift : No specific requirement for the generated code version
+
+## 1.Kitex Fallback Functional Description
+
+
+After an RPC request fails, businesses usually take some fallback measures to ensure a valid response (such as constructing a default response when encountering timeouts or circuit breakers). Kitex's Fallback supports handling all exceptional requests. Additionally, as business exceptions usually return through Resp (BaseResp), Fallback also supports handling Resp.
```
Comment (YangruiEmma) [inline content/en/docs/kitex/Tutorials/basic-feature/fallback.md:18]: 'business exceptions **usually** return through Resp' modify '**usually**' to '**maybe**'. 

'business exceptions **maybe** return through Resp

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md (hunk 5 above)
Comment (morpeko7) [reply content/en/docs/kitex/Tutorials/basic-feature/fallback.md]: Modified

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md (hunk 6 above)
Comment (morpeko7) [reply content/en/docs/kitex/Tutorials/basic-feature/fallback.md]: Modified

File: content/en/docs/kitex/Tutorials/basic-feature/fallback.md (hunk 7 above)
Comment (morpeko7) [reply content/en/docs/kitex/Tutorials/basic-feature/fallback.md]: Modified

**Pull request merged**